highscores.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/styles/styles.css" />
+    <title>Coding Quiz</title>
+  </head>
+  <!-- nav tag -->
+  <header class="header">
+    <!-- nav link home -->
+    <nav class="nav-container">
+      <a href="index1.html" class="nav-link"> Home </a>
+      <a href="./highscores.html" class="nav-link"> Highscores </a>
+    </nav>
+  </header>
+  <body>
+    <main id="high-score-main">
+      <!-- <section id="high-score-id" class="high-scores-section">
+        <h1 class="table-high-scores">Table Of Highscores</h1>
+        <div class="table-container">
+          <table class="table-scores">
+            <tr class="table">
+              <th class="table-heading-1">Name</th>
+              <th class="table-heading-2">Score</th>
+            </tr>
+            <tr class="table-result">
+              <td class="table-data-1"></td>
+              <td class="table-data-2"></td>
+            </tr>
+          </table>
+        </div>
+      </section> -->
```

Comment:
If code is no longer used, please remove it. Don't just comment it out as this looks messy